packages/tools/perftest/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
because this depends on GPL PCDs it may also have to be GPL

Reply:
I'm waffling on the right way to handle the licensing question.  I think it's valid for GPL-using code to be published under both GPL and MIT.  MIT is strictly more permissive so my intent here is that anyone can do anything with the code in this package, though they'll still have to follow the GPL's rules on the recursive dependencies (which is always required).  What I'm trying to not do is say that the code I wrote in this package can never be used outside of GPL.  IANAL so no idea if there's a clean way to say that.